packages/app/src/vis-packs/core/heatmap/HeatmapToolbar.tsx

```diff
@@ -73,7 +78,20 @@ function HeatmapToolbar(props: Props) {
 
       <Separator />
 
-      <FlipYAxisToggler value={flipYAxis} onToggle={toggleYAxisFlip} />
+      <ToggleBtn
+        label="X"
```

Comment:
Shortening the visible labels to just "X" and "Y", as the icon is quite explicit.